contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
+        altText="thumbnail image"
+        :appearanceOverrides="{
+          objectFit: thumbnailScaleType,
+          backgroundColor: $themeTokens.fineLine
+        }"
+        :style="KImgVerticalSmallStyle"
+      />
+      <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+      <div class="horizontal-layout-style">
+
+        <KImg
+          v-if="layout === 'horizontal' && thumbnailDisplay === 'large'"
```

Comment:
[Required] Can you move this condition to the `<div class="horizontal-layout-style">` line above? I don't think we need to render that div when it's not needed.